unit_tests/connector_builder/test_connector_builder_handler.py

```diff
@@ -32,6 +32,7 @@
     StreamReadPages,
     StreamReadSlices,
 )
+from airbyte_cdk.legacy.sources.declarative.declarative_stream import DeclarativeStream
```

Comment:
I think this can be removed. It was still used during the migration but I think it's not anymore